ibis/backends/duckdb/__init__.py

```diff
@@ -232,14 +233,16 @@ def read_json(
     ) -> ir.Table:
         """Read newline-delimited JSON into an ibis table.
 
+        !!! note "This feature requires duckdb>=0.7.0"
```

Comment:
Is there a way we could check that this is true on the connection, and raise a better error message if the `duckdb` version is too low? If this is hard to do don't worry about it.

Reply:
Yep, lemme give it a shot.

Reply:
Fixed. Let me know if the error message is sufficient.